styles/mixins/patterns/_tables.scss

```diff
@@ -38,56 +38,54 @@ $-min-row-height: var.$touch-target-size;
   th,
   td {
     @include layout.padding;
+    // Prevent extra vertical height with <th> elements
+    // FIXME: Review after typography patterns introduced
+    line-height: 1;
   }
 
   td {
     @include atoms.border(top);
   }
 
-  thead {
-    // Prevent extra vertical height with <th> elements
-    // FIXME: Review after typography patterns introduced
-    line-height: 1;
-
-    th {
-      @include containers.sticky-header;
-      border-bottom: 1px solid $-color-header-border;
-      background-color: $-color-header-background;
-      // Ensure the header is displayed above content in the table when it is
-      // scrolled, including any content which establishes a new stacking context.
-      z-index: 1;
-      text-align: left;
-    }
+  &__header {
+    @include containers.sticky-header;
+    border-bottom: 1px solid $-color-header-border;
+    background-color: $-color-header-background;
+    // Ensure the header is displayed above content in the table when it is
+    // scrolled, including any content which establishes a new stacking context.
+    z-index: 1;
+    text-align: left;
   }
 
   tbody {
     // Make table content look interact-able
     cursor: pointer;
+  }
+
+  tr {
+    height: $-min-row-height;
+    @include layout.padding;
+  }
+
+  // No border on top of first row's <td> elements, to eliminate a
+  // double border with the <th>s
+  tr:first-child td {
```

Comment:
Great question! I had to verify for my own self. Absolutely no difference in the end: the generated selector is the same in both cases:

`.Hyp-Table tr:first-child td`

The `&` is, strictly speaking, redundant here, so I'd propose that our convention be to only use the `&` when it does make a difference. It would be necessary in a case like:

```
.MyThing {
  &.foo { }
  &__bar { }
}
```

Which would create selectors for `.MyThing.foo` (not a terribly-common pattern in our world, but we do use it for state classes, e.g. `.MyThing.is-selected`) and `.MyThing__bar` (which is a pattern we use frequently).